Specification.md

```diff
@@ -40,9 +40,11 @@ seq=11                                  \
 
 ## 2. Format and Naming
 
-**Sync Interest Format**: `/<group-prefix>/<state-vector>/<signature>`
+**Sync Interest Format**: `/<group-prefix>/v=2/<signature>`
```

Comment:
Maybe this should say "Name" rather than "Format"?

Also, reminder to check/fix the signature-in-the-interest-name issue that came up yesterday.

Reply:
If this refers to Signed Interest v0.3 format, it needs to specify which of SigTime/SigNonce/SigSeqNum are required and which ones are discouraged.

Reply:
> If this refers to Signed Interest v0.3 format, it needs to specify which of SigTime/SigNonce/SigSeqNum are required and which ones are discouraged.

None. We need to change the signed interest format

Reply:
Actually we don't, it's only a recommendation.